src/rest/SearchEndpoint.ts

```diff
@@ -80,7 +80,8 @@ export class SearchEndpoint implements ISearchEndpoint {
         _.extend(
           {
             restUri: 'http://localhost:8100/rest/search',
-            searchAlertsUri: 'http://localhost:8088/rest/search/alerts/'
+            searchAlertsUri: 'http://localhost:8088/rest/search/alerts/',
+            accessToken: '52d806a2-0f64-4390-a3f2-e0f41a4a73ec'
```

Comment:
It's the token for the v7 searchuisamples org. I added it so that when we add the local setup argument, it works by default. It looks like the search api changed its local setup to use v8 so I'll just remove this.